src/templates/post/Post.js

```diff
@@ -20,20 +16,25 @@ const Post = ({ imageUrl, tags, date, readingTime, title }) => (
       readingTime={readingTime}
       title={title}
     />
-    <PostContent>
-      <Paragraph strong>
-        Bacon ipsum dolor amet chislic filet mignon cow, spare ribs short loin
-        beef ribs pork chop. Tail frankfurter ribeye pork chop pig rump short
-        ribs pork bresaola tongue shoulder jerky alcatra jowl. Boudin swine
-        filet mignon, fatback kielbasa leberkas pork belly pig salami alcatra
-        doner prosciutto t-bone. Ham bacon prosciutto, pork belly turkey tri-tip
-        pancetta. Alcatra tenderloin t-bone buffalo beef ribs fatback burgdoggen
-        spare ribs pork loin pork chop salami capicola hamburger pork drumstick.
-        Drumstick jerky jowl, short ribs chislic turducken burgdoggen kevin
-        venison shoulder pancetta cupim tongue meatball ham.
+    <Box width="xlarge">
```

Comment:
If you are using several tags (parent and children) to represent the text parts, you will need to create a separate component for it, say `<P>`, or `<Paragraph>` but of our own (which re-used grommet's). The reason for it is that Gatsby parses markdown into MDX AST and it makes paragraphs which normally are represented as simple `<p>` HTML tags. In Gatsby, you can re-assign these basic HTML representations to specific components. Like, you can say "whenever it is a `<p>` in my text, use my React component `<P>` instead. But for that, you need to have these components prepared.

Reply:
Done, added <Paragraph> component